Cargo.toml

```diff
@@ -128,5 +128,5 @@ sprockets-rot = { git = "https://github.com/oxidecomputer/sprockets.git", defaul
 stm32g0 = { git = "https://github.com/oxidecomputer/stm32-rs-nightlies", branch = "stm32g0b1-update", default-features = false }
 tlvc = { git = "https://github.com/oxidecomputer/tlvc", default-features = false }
 tlvc-text = { git = "https://github.com/oxidecomputer/tlvc", default-features = false }
-transceiver-messages = { git = "https://github.com/oxidecomputer/transceiver-control", default-features = false }
+transceiver-messages = { path = "../transceiver-control/messages", default-features = false }
```

Comment:
This will get backed out to point at git again once https://github.com/oxidecomputer/transceiver-control/pull/34 is merged and I don't need to keep switching it to test locally.